packages/create-permissionless-app/template/base/src/components/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from "next/image"
+import React from "react"
+
+export const Header: React.FC = () => {
+    return (
+        <div className="z-10 max-w-5xl w-full items-center justify-between font-mono text-sm lg:flex">
+            <p className="fixed left-0 top-0 flex flex-col w-full justify-center border-b border-gray-300 bg-gradient-to-b from-zinc-200 pb-6 pt-8 backdrop-blur-2xl dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit lg:static lg:w-auto  lg:rounded-xl lg:border lg:bg-gray-200 lg:p-4 lg:dark:bg-zinc-800/30">
+                Get started by cloning&nbsp;
```

Comment:
I think we might want to change this, as this template shouldn't need to clone anything.

Reply:
yeah sure.. i just copied the template code from one of our starter kit..